### PR TITLE
feat: save colapsed sider status in local storage

### PR DESCRIPTION
**Pull request by valerydluski**
**🟢 Add `deploy` label if you want to deploy this Pull Request to staging environment**

#### 🧑‍⚖️ Pull Request Naming Convention

- Title should follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- Do not put issue id in title
- Do not put WIP in title. Use [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) functionality
- Consider to add `area:*` label(s)

* [x] I followed naming convention rules

---

#### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Test Case
- [ ] Documentation update
- [ ] Other

#### 🔗 Related issue link

_Describe the source of requirement, like related issue link._

#### 💡 Background and solution

_Describe the big picture of your changes here_

#### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Database migration is added or not needed
- [x] Documentation is updated/provided or not needed
- [x] Changes are tested locally

### CODE REVIEW

File: client/src/components/Sider/AdminSider.tsx [hunk 1]
```diff
@@ -4,15 +4,20 @@ import SubMenu from 'antd/lib/menu/SubMenu';
...
+import { useMemo } from 'react';
+import { useLocalStorage } from 'react-use';
 import { Course } from 'services/models';
 import { getAdminMenuItems, getCourseManagementMenuItems } from './data/menuItems';
 const { Sider } = Layout;
 
 type Props = { session: Session; courses: Course[]; activeCourse?: Course | null };
 
+enum LocalStorage {
+  isCollapsedSider = 'isCollapsedSider',
```
Comment (mikhama) [inline client/src/components/Sider/AdminSider.tsx:16]: IsSliderCollapsed

File: client/src/components/Sider/AdminSider.tsx [hunk 2]
```diff
@@ -4,15 +4,20 @@ import SubMenu from 'antd/lib/menu/SubMenu';
...
+}
+
 export function AdminSider(props: Props) {
-  const [collapsed, setCollapsed] = useState(true);
+  const [isCollapsedSider = true, setIsCollapsedSider] = useLocalStorage<boolean>(LocalStorage.isCollapsedSider);
```
Comment (mikhama) [inline client/src/components/Sider/AdminSider.tsx:20]: isSliderCollapsed

File: client/src/components/Sider/AdminSider.tsx (hunk 1 above)
Comment (valerydluski) [reply client/src/components/Sider/AdminSider.tsx]: done

File: client/src/components/Sider/AdminSider.tsx (hunk 2 above)
Comment (valerydluski) [reply client/src/components/Sider/AdminSider.tsx]: done

**Review by mikhama — APPROVED** [review]

**Pull request merged**